docker-compose.override.yml

```diff
@@ -15,8 +15,25 @@ services:
   web:
     build:
       target: development
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      - MINIO_HOST=localhost:9000
+      - MINIO_ACCESS_KEY=normal-oj-testing
+      - MINIO_SECRET_KEY=normal-oj-testing-secret
+      - MINIO_BUCKET=normal-oj-testing
     volumes:
       - ./Back-End:/app
+  minio:
+    image: quay.io/minio/minio
+    network_mode: "service:web"
```

Comment:
why don't you use default bridge mode here?

Reply:
I tried but I didn't find a way that is working. 

This is for backend to access minio via localhost:9000, in order to make the front-end code getting the presigned urls with localhost:9000 as well, instead of minio:9000.

Reply:
got it. Maybe we can add a comment to describe the reason.

Reply:
no problem, fixed.